superset-frontend/cypress-base/cypress/e2e/chart_list/list.test.ts

```diff
@@ -112,9 +112,10 @@ describe('Charts list', () => {
       cy.getBySel('sort-header').eq(1).contains('Name');
       cy.getBySel('sort-header').eq(2).contains('Type');
       cy.getBySel('sort-header').eq(3).contains('Dataset');
-      cy.getBySel('sort-header').eq(4).contains('Owners');
-      cy.getBySel('sort-header').eq(5).contains('Last modified');
-      cy.getBySel('sort-header').eq(6).contains('Actions');
+      cy.getBySel('sort-header').eq(4).contains('Dashboards added to');
```

Comment:
This is maybe out of scope, but I feel `Dashboards added to` is too verbose, and I think just going with `Dashboards` or `On dashboards` makes more sense.

Reply:
@yousoph @kasiazjc wdyt?

Reply:
@kasiazjc says that "On dashboards" sounds fine. I changed it here (in charts list) and also in Explore for consistence